sklearn/linear_model/tests/test_ridge.py

```diff
@@ -694,3 +694,19 @@ def test_n_iter():
         reg = Ridge(solver=solver, max_iter=1, tol=1e-1)
         reg.fit(X, y_n)
         assert_equal(reg.n_iter_, None)
+
+
+def test_ridge_fit_intercept_sparse():
+    r = np.random.RandomState(42)
+    X = r.randn(1000, 2)
+    w = r.randn(2)
+    i = 10
+    y = np.dot(X, w) + i
+    X_csr = sp.csr_matrix(X)
+
+    dense = Ridge(alpha=1., tol=1.e-15, solver='sag', fit_intercept=True)
```

Comment:
is there a check that the solution to the dense model doesn't depend on the solver? (I hope so).